lib/beaker/hypervisor/docker.rb

```diff
@@ -102,7 +113,7 @@ def provision
         fix_ssh(container) if @options[:provision] == false
 
         @logger.debug("Starting container #{container.id}")
-        container.start({"PublishAllPorts" => true, "Privileged" => true})
```

Comment:
I'm still a bit unclear why removing the parameters has this effect; the `start` method only makes an api call with parameters if they are provided, and doesn't seem to do any sort of reading of currently set parameters from the logic seen [here](https://github.com/swipely/docker-api/blob/master/lib/docker/container.rb#L201-L204).